File: src/exemple/item_repository.ts
import { SyncalbeRepository } from "../core/implementation/data/syncable_object_repository";
import { Item } from "./item";
import { ItemDataSource } from "./item_datasource";


export class ItemRepository  extends SyncalbeRepository<Item> {
    constructor() {
        super(new ItemDataSource(),Item);
    }
}

File: fast_sync_dotnet_service/exemple/Item/ItemRepository.cs
using fast_sync_core.abstraction.data;
using fast_sync_core.implementation.data;

namespace exemple.Item
{
    public class ItemRepository : SyncableRepository<Item>
    {
        public ItemRepository(ISyncableDataSource<Item> dataSource) : base(dataSource)
        {
        }
    }
}
